site-src/v1alpha2/guides/getting-started.md

```diff
@@ -36,7 +36,7 @@ these resources. Installing the CRDs will just allow you to see and apply the
 resources, though they won't do anything.
 
 ```
-kubectl kustomize "github.com/kubernetes-sigs/gateway-api/config/crd?ref=v0.3.0" \
+kubectl kustomize "github.com/kubernetes-sigs/gateway-api/config/crd?ref=v0.4.0-rc1" \
```

Comment:
This will ensure that only v1alpha2 CRDs are installed and avoid any confusing kubectl get conflicts.
```suggestion
kubectl kustomize "github.com/kubernetes-sigs/gateway-api/config/crd/v1alpha2?ref=v0.4.0-rc1" \
```

Reply:
Thank you! But it seems https://github.com/kubernetes-sigs/gateway-api/tree/master/config/crd/v1alpha2 does not have `kustomization.yaml` so we cannot run the command against the directory. Here is the error output (w/ kubectl v1.21):

```
$ kubectl kustomize "github.com/kubernetes-sigs/gateway-api/config/crd/v1alpha2?ref=v0.4.0-rc1"

Error: unable to find one of 'kustomization.yaml', 'kustomization.yml' or 'Kustomization' in directory '/tmp/kustomize-263870027/config/crd/v1alpha2'
```

And the https://github.com/kubernetes-sigs/gateway-api/blob/master/config/crd/kustomization.yaml defines only `v1alpha2` resources so `kubectl kustomize "github.com/kubernetes-sigs/gateway-api/config/crd?ref=v0.4.0-rc1"` would be fine?

Reply:
🤦 you're completely right, thanks!